client-app/src/mobile/popover/PopoverPage.js

```diff
@@ -71,16 +70,11 @@ const popoverContent = hoistCmp.factory({
 /**
  * LocalModel used to demonstrate opening a Popover in controlled mode
  */
-class LocalModel extends HoistModel {
-    @observable isOpen = false;
+class PopoverPageModel extends HoistModel {
+    @bindable isOpen = false;
```

Comment:
wondering about this choice. Thought the idea was to only used bindables for cases where we use the hoist "bind" magic.

Reply:
we are using it on line 36 above from the popover/